tests/network/hotplug_sriov.go

```diff
@@ -49,7 +49,9 @@ var _ = SIGDescribe("[Serial] SRIOV nic-hotplug", Serial, decorators.SRIOV, func
 	sriovResourceName := readSRIOVResourceName()
 
 	BeforeEach(func() {
-		Expect(checks.HasFeature(virtconfig.HotplugNetworkIfacesGate)).To(BeTrue())
+		if !checks.HasFeature(virtconfig.HotplugNetworkIfacesGate) {
+			Skip("HotplugNICs feature gate is disabled.")
+		}
```

Comment:
Make sense to move this please after the BeforeEach of line 57
(first to check setup even has SRIOV enabled)

Reply:
I tend to agree, but I think it should be done in a follow-up PR.

Reply:
I think it is such small change, that even another commit fits, but whatever you prefer
We can save time of lanes, bump and so on, also review time

Reply:
The suggested change will not affect the problem we want to quickly fix (PRs are blocked in kubevirt/kubevirtci).

Reply:
I know, but my point is different, it is a cosmetic logical that can be done in a jiffy, and can save resources
but from here it is yours

Reply:
It is approved.

Reply:
a change doesnt clean the approve, this what i meant